app/mailers/concerns/autoresponder.rb

```diff
@@ -0,0 +1,9 @@
+module Autoresponder
+  include ActiveSupport::Concern
+
+  def autorespond(parsed_email)
+    mail(from: noreply_address,
+         to: parsed_email.from.to_s,
+         subject: 'This mailbox is not monitored')
+  end
```

Comment:
I don't think so.  It's an artefact of me trying to pull all the existing logic from the old app.  I'll drop it.